Cargo.toml

```diff
@@ -21,3 +21,8 @@ crate-type = ["cdylib", "rlib"]
 
 [dependencies]
 wai-bindgen-rust = "0.2.2"
+original = { version = "0.8.2", package = "sgp4" }
+chrono = "0.4.23"
+
+[patch.crates-io]
+original = { path = "../../Rust/sgp4", package = "sgp4" }
```

Comment:
You'll need to update this to use `git` dependencies. At the moment, this won't build in CI or on my computer because I don't have your fork of `sgp4` checked out locally.

Reply:
@dynamite-bud we can't merge this PR until you resolve this.